adal/src/main/java/com/microsoft/aad/adal/DeserializationAuthenticationException.java

```diff
@@ -46,13 +39,4 @@ public DeserializationAuthenticationException(String msg) {
         super(defaultErrorCode, msg);
     }
 
-    /**
-     * Constructs a new DeserializationAuthenticationException with message and the cause exception
-     * @param details Details related to the error such as query string, request
-     *            info
-     * @param throwable {@link Throwable}
-     */
-    public DeserializationAuthenticationException(String msg, Throwable cause) {
-        super(defaultErrorCode, msg, cause);
-    }
```

Comment:
yes

---
In reply to: [65640763](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65640763) [](ancestors = 65640763)